python/src/energuide/snippets.py

```diff
@@ -27,7 +27,7 @@ class _HouseSnippet(typing.NamedTuple):
     heated_floor_area: str
     heating_cooling: str
     ventilation: typing.List[str]
-    water_heating: typing.List[str]
+    water_heating: str
```

Comment:
After updating to `Element` I started getting errors that led me here. Is this right?

Reply:
That is correct, that is odd because I thought I had already fixed that